eclipse.platform.common/bundles/org.eclipse.platform.doc.isv/reference/misc/runtime-options.html

```diff
@@ -115,6 +115,10 @@ <h2><a name="commandline"></a>Command line arguments </h2>
   <dd>If specified the executable will not display any error or message
 dialogs.&nbsp; This is useful if the executable is being used in an
 unattended situation.</dd>
+  <dt>--launcher.noRestart (Executable)</dt>
+  <dd>If specified, the special handling of exit codes 23 (restart) and 24
+(relaunch) is disabled, treating them like regular exit codes. This is useful
```

Comment:
I'm not a native English-speaker so that impression might be wrong, but I'm not sure if the second half of the first sentence is correct?
Furthermore I think the second sentence should be a bit more defensive. 
```suggestion
(relaunch) is disabled and they are treated as regular exit codes. This can be useful
```

Reply:
I applied it, with rewrapping to 80 character lines, as a forced pushed update to the existing commit.